plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenFacetImpl.java

```diff
@@ -251,9 +270,25 @@ public Content put(final StorageTx tx, final MavenPath path, final Payload paylo
       throws IOException, InvalidContentException
   {
     log.debug("PUT {} : {}", getRepository().getName(), path.getPath());
```

Comment:
since the new method also logs this and this is basically a forward to the new method, the above logging line should be removed otherwise you'll get duplicate entries